scale_build/image/utils.py

```diff
@@ -1,15 +1,45 @@
+import json
 import os
+import shutil
 
+from scale_build.exceptions import CallError
 from scale_build.utils.environment import APT_ENV
 from scale_build.utils.run import run
-from scale_build.utils.paths import CHROOT_BASEDIR
+from scale_build.utils.paths import CHROOT_BASEDIR, TMP_DIR
+
+from .manifest import UPDATE_FILE
 
 
 PACKAGE_PATH = os.path.join(CHROOT_BASEDIR, 'packages')
+IMAGE_VERSION = None
 
 
 def run_in_chroot(command, logger=None, exception_message=None, **kwargs):
     return run(
         ['chroot', CHROOT_BASEDIR] + command, logger=logger, exception_msg=exception_message,
         env={**APT_ENV, **os.environ}, **kwargs
     )
+
+
+def get_image_version():
+    global IMAGE_VERSION
+    if IMAGE_VERSION:
+        return IMAGE_VERSION
+
+    if not os.path.exists(UPDATE_FILE):
+        raise CallError(f'{UPDATE_FILE!r} update file does not exist')
+
+    mount_dir = os.path.join(TMP_DIR, 'update_dir')
+    if os.path.exists(mount_dir):
+        shutil.rmtree(mount_dir)
```

Comment:
Can you just mount it under a temporary directory without removing files from filesystem and introducing unnecessary race conditions?